utils/src/jsMain/kotlin/com/badoo/reaktive/utils/Timers.kt

```diff
@@ -0,0 +1,42 @@
+package com.badoo.reaktive.utils
+
+import kotlin.browser.window
+
+external fun setTimeout(handler: dynamic, timeout: Int = definedExternally, vararg arguments: Any?): Int
+external fun setInterval(handler: dynamic, timeout: Int = definedExternally, vararg arguments: Any?): Int
+external fun clearTimeout(handle: Int = definedExternally)
+external fun clearInterval(handle: Int = definedExternally)
+
+object Timers {
+    fun setTimeout(delayMillis: Long, task: () -> Unit, vararg arguments: Any?): Int {
+        return try {
+            window.setTimeout(task, delayMillis.toInt(), arguments)
+        } catch (ex: dynamic) {
+            setTimeout(task, delayMillis.toInt(), arguments)
```

Comment:
Avoiding clashing using different types is a bad workaround. Let's try the following:
1. Put `external` functions into `external` package
2. Import them into this file using named alias, e.g. `import xxx.external.setTimeout as setTimeoutExternal`
3. Remove `Timers` object and make these functions `internal` and top-level
3. Use `Int`

Reply:
@arkivanov I put those functions in `utils` module, so if I change them to internal I cannot reference from `reaktive` module. Would you like to not make them internal and keep in utils? OR should I move them to `reaktive` module and make them internal?

Reply:
They are needed in the `reaktive` module so let's put them there (in `jsMain` source set) and make `internal`.